### PR TITLE
[24] Targets update

### PR DESCRIPTION
**Pull request by juanmanuelramallo**
- Resolves: https://trello.com/c/Dv4rDyVO/24-targets-update and https://trello.com/c/XddKKlaC/25-notifications-after-target-update

### CODE REVIEW

File: app/models/target.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@ class Target < ApplicationRecord
   validates :area_length, numericality: { greater_than: 0 }
   validate :user_must_have_less_than_maximum_targets
 
-  after_create :broadcast_to_compatible_users
+  after_save :broadcast_to_compatible_users
```
Comment (GuilleLeopold) [inline app/models/target.rb]: Do you want to broadcast to all compatible users everytime you change something in a target?

Comment (juanmanuelramallo) [reply app/models/target.rb]: Ahhmmmm good catch. No 😢 
I just want to broadcast after an update to the new compatible targets.
But that would be a bit harder, wouldn't it?

File: app/controllers/targets_controller.rb [hunk 2]
```diff
@@ -9,12 +9,16 @@ def create
...
+  end
+
   def index
     render json: current_user.targets
   end
 
-  def destroy
-    render json: target.destroy
+  def update
+    render json: target, status: :accepted if target.update!(target_params)
```
Comment (Alxzu) [inline app/controllers/targets_controller.rb]: What would be the result here if it can't be updated?

Comment (juanmanuelramallo) [reply app/controllers/targets_controller.rb]: It'd be this https://github.com/juanmanuelramallo/target-mvd/blob/master/app/controllers/concerns/common_errors_concern.rb#L27

**Pull request merged**